localisation/00_wc_random_artifact_names.csv

```diff
@@ -21,60 +37,60 @@ artifact_rare_origin;Fables tell that [GetArtifactRareOrigin].;Des fables disent
+artifact_rare_origin_9;[GetArtifactThisThese] [This.GetName] [GetArtifactWasWere] found in the process of plundering the depths of an ancient tomb;ce [This.GetName] a �t� trouv� pendant le pillage des profondeurs d'une ancienne tombe;;;;;;;;;;;;x
+artifact_rare_origin_10;a tyrant, who took enormous pride in his appearance, was wearing [GetArtifactThisThese] [This.GetName] the day he was murdered by his own servants;un tyran, qui �tait tr�s fier de son apparence, portait ce [This.GetName] le jour o� il a �t� assassin� par ses propres serviteurs;;;;;;;;;;;;x
+;;;;;;;;;;;;;;x
+artifact_epic_origin_1;[GetArtifactThisThese] [This.GetName] fell from the sky;ce [This.GetName] est tomb� du ciel;;;;;;;;;;;;x
+artifact_epic_origin_2;the origins of [GetArtifactThisThese] [This.GetName] are unknown;les origines de ce [This.GetName] sont inconnues;;;;;;;;;;;;x
+artifact_epic_origin_3;[GetArtifactThisThese] [This.GetName] [GetArtifactWasWere] the spark that ignited full-scale war between neighbouring realms, as they vied to acquire [GetArtifactItThem];ce [This.GetName] a �t� l'�tincelle qui a d�clench� une guerre � grande �chelle entre des royaumes voisins, alors qu'ils rivalisaient pour l'acqu�rir;;;;;;;;;;;;x
+artifact_epic_origin_4;[GetArtifactThisThese] [This.GetName] [GetArtifactIsAre] the only remaining proof of a mystic creature's existence, which died a long time ago;ce [This.GetName] est la seule preuve restante de l'existence d'une cr�ature mystique morte il y a longtemps;;;;;;;;;;;;x
+artifact_epic_origin_5;[GetArtifactThisThese] [This.GetName] [GetArtifactWasWere] sold once for 40000;ce [This.GetName] a �t� vendu une fois pour 40 000 pi�ces d'or;;;;;;;;;;;;x
+artifact_epic_origin_6;the original owner of [GetArtifactThisThese] [This.GetName] spent the last years of his life traveling the world in search of esoteric secrets;le premier propri�taire de ce [This.GetName] a pass� les derni�res ann�es de sa vie � parcourir le monde � la recherche de secrets �sot�riques;;;;;;;;;;;;x
+artifact_epic_origin_7;sometimes [GetArtifactThisThese] [This.GetName] glow[GetArtifactEndingSingularS] with the essence of several lives that ended too soon;parfois ce [This.GetName] brille avec l'essence de plusieurs vies qui se sont termin�es trop t�t;;;;;;;;;;;;x
+artifact_epic_origin_8;when you gaze too long at [GetArtifactThisThese] [This.GetName], [GetArtifactItThey] also gaze[GetArtifactEndingPluralS] back at you;quand vous regardez trop longtemps ce [This.GetName], il vous regarde aussi en retour;;;;;;;;;;;;x
+artifact_epic_origin_9;[GetArtifactThisThese] [This.GetName] whisper[GetArtifactEndingPluralS] [GetArtifactItsTheir] original owner's name from realms beyond;ce [This.GetName] chuchote le nom de son propri�taire d'origine depuis des royaumes lointains;;;;;;;;;;;;x
```

Comment:
Should be whispers.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/89428268-464b1200-d74d-11ea-8614-245d91f9bd44.png)

</details>

Reply:
Should be fine now

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/89439433-dcce0200-d752-11ea-8726-d4b21d992eda.png)
![image](https://user-images.githubusercontent.com/46576860/89439497-f53e1c80-d752-11ea-887f-bac8e11843de.png)

</details>

Reply:
looks good